docs/src/content/docs/ja/reference/icons.mdx

```diff
@@ -3,17 +3,16 @@ title: アイコン
 description: Starlightで利用可能なすべてのアイコンの概要。
 ---
 
-Starlightには、`<Icon>`コンポーネントを使用してコンテンツに表示できる組み込みのアイコンセットが用意されています。
+Starlightには、`<Icon>`コンポーネントを使用してコンテンツに表示できる、組み込みのアイコンセットが用意されています。
 
 ## アイコンの使用
 
-アイコンは[`<Icon>`](/ja/components/icons/)コンポーネントを使用して表示できます。
-また、[カード](/ja/components/cards/)や[ヒーローアクション](/ja/reference/frontmatter/#hero)などの設定のような他のコンポーネントでもよく使用されます。
+アイコンは[`<Icon>`](/ja/components/icons/)コンポーネントを使用して表示できます。また、[カード](/ja/components/cards/)や[ヒーローアクション](/ja/reference/frontmatter/#hero)の設定など、他のコンポーネントでもよく使用されます。
```

Comment:
Although not part of the changes in https://github.com/withastro/starlight/pull/2505, I have fine-tuned them slightly for better readability.